.grit/patterns/openai_v4.md

```diff
@@ -115,14 +115,64 @@ pattern change_completion_try_catch() {
     }
 }
 
+pattern change_imports() {
+    any {
```

Comment:
This `any` clause looks redundant.

Reply:
@nsbradford Why did you add an empty any? An any with a single nested entry is useless.